### PR TITLE
Added launch settings in Advanced Tuning tab + nav_fw_control_smoothness

### PR DESCRIPTION
**Pull request by pablitous**
Now its working without any new msp message. just like OSD tab.
Already tested and working.

Also i have a question.
Im planing to add also "nav_fw_control_smoothness", is it just for fixed wing? can i add it in advanced tuning tab in fixed wing navigation settings section?

EDIT: also added gps_ublox_use_galielo on configuration tab.

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -758,6 +758,81 @@
     "configurationBatteryCapacityUnit": {
         "message": "Battery Capacity Unit"
     },
+    "configurationLaunch": {
+        "message": "Auto Launch Configuration"
```
Comment (giacomo892) [inline _locales/en/messages.json:762]: I'd change this into:
Fixed Wing Auto Launch Settings

File: _locales/en/messages.json [hunk 2]
```diff
@@ -758,6 +758,81 @@
...
+    },
+    "configurationLaunchVelocity": {
+        "message": "Velocity"
```
Comment (giacomo892) [inline _locales/en/messages.json:765]: Threshold Velocity 
looks better

File: _locales/en/messages.json [hunk 3]
```diff
@@ -758,6 +758,81 @@
...
+    },
+    "configurationLaunchAccel": {
+        "message": "Acceleration"
```
Comment (giacomo892) [inline _locales/en/messages.json:768]: Threshold Acceleration

File: _locales/en/messages.json [hunk 4]
```diff
@@ -758,6 +758,81 @@
...
+        "message": "Throttle"
+    },
+    "configurationLaunchIdleThr": {
+        "message": "Idle Throttle"
+    },
+    "configurationLaunchMotorDelay": {
+        "message": "Motor Delay"
+    },
+    "configurationLaunchSpinupTime": {
+        "message": "Spinup Time"
```
Comment (giacomo892) [inline _locales/en/messages.json:786]: Motor Spinup Time

File: _locales/en/messages.json [hunk 5]
```diff
@@ -758,6 +758,81 @@
...
+        "message": "Auto Launch Configuration"
+    },
+    "configurationLaunchVelocity": {
+        "message": "Velocity"
+    },
+    "configurationLaunchAccel": {
+        "message": "Acceleration"
+    },
+    "configurationLaunchMaxAngle": {
+        "message": "Max Angle"
```
Comment (giacomo892) [inline _locales/en/messages.json:771]: Max Tilt Angle

File: _locales/en/messages.json [hunk 6]
```diff
@@ -758,6 +758,81 @@
...
+        "message": "Min Time"
+    },
+    "configurationLaunchTimeout": {
+        "message": "Time Out"
+    },
+    "configurationLaunchMaxAltitude": {
+        "message": "Max Altitude"
+    },
+     "configurationLaunchClimbAngle": {
+        "message": "Climb Angle"
```
Comment (moggiex) [inline _locales/en/messages.json:798]: "Climb angle for launch sequence (degrees), is also restrained by global max_angle_inclination_pit" also see the other descriptions from here  [https://github.com/iNavFlight/inav/blob/56be63d5f1f220b4f650ec448f4e5d4d729e951f/src/main/fc/settings.yaml](https://github.com/iNavFlight/inav/blob/56be63d5f1f220b4f650ec448f4e5d4d729e951f/src/main/fc/settings.yaml)

File: tabs/advanced_tuning.js [hunk 7]
```diff
@@ -124,9 +107,39 @@ TABS.advanced_tuning.initialize = function (callback) {
         $('#advanced-tuning-save-button').click(function () {
             saveChainer.execute();
         });
-
+        
+        $('a.save').click(function () {
+            Settings.saveInputs().then(function () {
+                var self = this;
+                MSP.promise(MSPCodes.MSP_EEPROM_WRITE);
+                GUI.log('OSD settings saved');
```
Comment (fiam) [inline tabs/advanced_tuning.js:115]: Please, change this log message to something more relevant, or alternatively, just remove it entirely.

File: _locales/en/messages.json (hunk 6 above)
Comment (pablitous) [reply _locales/en/messages.json:798]: i dont get what you are trying to say. I already added that HELP description when you mouseover the help.
would you like to change that value also just like the others i add?

File: tabs/advanced_tuning.js (hunk 7 above)
Comment (pablitous) [reply tabs/advanced_tuning.js:115]: yea. just remove it.

**Pull request merged**